src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-unused-vars */
+import { Redux } from 'react';
+import books from './books/books';
+import categories from './categories/categories';
```

Comment:
- [ ] Ohhh, everything is good there but you forgot only one thing :smile: , you know what it is? :thinking: 
     - you forgot to install the `@reduxjs/toolkit` and Import it in your `configureStore` as it is [required](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Add%20Redux%20Toolkit(npm%20install%20react%2Dredux%20%40reduxjs/toolkit).) :+1: 

**Note** : This [link](https://redux-toolkit.js.org/introduction/getting-started) can help you to install the `@reduxjs/toolkit` and how to use it. :smiley: